configuration/concepts/newEpisodesOfDiseasesReportConcepts.csv

```diff
@@ -0,0 +1,9 @@
+uuid,Void/Retire,Fully specified name:en,Description:en,Data class,Data type,Answers,Members,Same as mappings,Short name:en,Fully specified name:fr,Short name:fr,Absolute high,Critical high,Normal high,Absolue low,Critical low,Normal low,Units,Display precision,Allow decimals,Complex data handler
+b290c073-7a2b-4667-8922-8a84af07aadf,,Intestinal parasitosis,,ConvSet,N/A,,Acute amoebic dysentery (A06.0);Ascariasis with intestinal complications (B77.0);Enterobiasis (B80);Giardiasis [lambliasis] (A07.1);Intestinal Parasites (B82.9),,,Parasitose intestinale,,,,,,,,,,,

```

Comment:
Could we investigate if the report shows the Short Name of this concept (if we provide one) and in which case, we should rename the FSN to be "MSPP New Episodes of Diseases - Intestinal parasitosis"

Reply:
And apply the same pattern for the other groupings.

Reply:
I'm afraid this would need considerable amount of rework to explicitly use the short name. This is because the report relies on the preferred name.

Reply:
I am not sure to follow. I'll reach ou on Slack about this.